### PR TITLE
Adds SurveyCompletion Screen

### PR DESCRIPTION
**Pull request by Ecarrion**
# Why
As a part of #2544, this PR adds the VC in charge of showing a completion screen after the user has finished submitting the in-app feedback survey.

Note: Navigation from Survey screen to Completion screen will be done in a different PR.

# How

- Adds `SurveyCompletedViewController.swift` and `SurveyCompletedViewController.xib`. It's a stack view based view to distributing most components with uniformity. A scroll view has been added to support scroll in when a user chooses a large accessibility class.

- Adds a new(missing?) `callout` style helper to apply to labels. 

- Launches Zendesk's new request screen after tapping in "contact us" button. 

- Adds two new image assets.

# Screenshots

Light | Dark | Large
--- | --- | ---
light | dark | Large font

# Testing Steps

- Add the following code:

In `ViewDidload`
```swift
let leftBarButton = UIBarButtonItem(barButtonSystemItem: .action, target: self, action: #selector(showSurvey))
navigationItem.leftBarButtonItem = leftBarButton
```

After `ViewDidload`
```swift
@objc private func showSurvey() {
    let vc = SurveySubmittedViewController()
    let navigationController = WooNavigationController(rootViewController: vc)
    present(navigationController, animated: true, completion: nil)
}
```

- See the survey completed screen property rendered.
- Test that tapping on the "contact us" button fires the new Zendesk issue flow.
- Test that tapping on the "back to store" button dismisses the screen.

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Resources/Images.xcassets/tablet-success.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "tablet-success-light.pdf",
```
Comment (jaclync) [inline WooCommerce/Resources/Images.xcassets/tablet-success.imageset/Contents.json:4]: just curious, why is this drawing called "tablet-success"? I just couldn't guess what this is from the name 😅  also, do we have an equivalent image for dark mode?

File: WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveySubmittedViewControllerTests.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import XCTest
+
+@testable import WooCommerce
+
+final class SurveySubmittedViewControllerTests: XCTestCase {
+
+    func testWhenTappingContactUsButtonThenTheContactUsPageIsPresented() throws {
```
Comment (jaclync) [inline WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveySubmittedViewControllerTests.swift:7]: wow, very nice test! 💯

File: WooCommerce/Resources/Images.xcassets/tablet-success.imageset/Contents.json (hunk 1 above)
Comment (Ecarrion) [reply WooCommerce/Resources/Images.xcassets/tablet-success.imageset/Contents.json:4]: It was called that way on Figma, apparently what the lady is holding is a tablet.
And there isn't a dark version in the designs, but I'll make sure to [ask G if one is needed](https://github.com/woocommerce/woocommerce-ios/issues/2544#issuecomment-667105875).

File: WooCommerce/Classes/ViewRelated/Survey/SurveySubmittedViewController.swift [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        configureStackViewsAxis()
+    }
+}
+
+// MARK: Constants
+//
+private extension SurveySubmittedViewController {
+    enum Localization {
+        static let title = NSLocalizedString("Feedback Sent!", comment: "Title in the navigation bar when the survey is completed")
+        static let thanks = NSLocalizedString("Thank you for sharing your\nthoughts with us", comment: "Text thanking the user when the survey is completed")
```
[the remark this answers is not in the document]
Comment (Ecarrion) [reply WooCommerce/Classes/ViewRelated/Survey/SurveySubmittedViewController.swift]: 👍 fixed in https://github.com/woocommerce/woocommerce-ios/pull/2584/commits/4481a9fd5c32a3e23fe0da580feff36bda82de31

**Comment by Ecarrion** [comment]
Thanks for the review @jaclync, I've made the changes according to your suggestions 🙇

**Pull request merged**